bin/start_fe.sh

```diff
@@ -33,7 +33,7 @@ eval set -- "$OPTS"
 RUN_DAEMON=0
 HELPER=
 HOST_TYPE=
-ENABLE_DEBUGGER=
+ENABLE_DEBUGGER=0
```

Comment:
seems we can drop the echo in line 48 too

Reply:
done